src/Polly.Strategies/Outcome.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Polly;
+
+public struct Outcome<T>
+{
+    public Outcome(T result) : this()
+    {
+        Result = result;
+    }
+
+    public Outcome(Exception? exception) : this()
+    {
+        Exception = exception;
+        Result = default!;
+    }
+
+    public T Result;
```

Comment:
Should this be a readonly property, instead of a read/write field?

Reply:
Yeah, my laziness. Fixed.